common/utils/category_resolver.py

```
@@ -0,0 +1,43 @@
+from typing import Dict
+
+from API.models import Insult
+from loguru import logger
+from prometheus_client import Histogram
+
+
+class Resolver:
+    cat_resolver_metric = Histogram(
+        "category_resolver", "Metric of the Duration of categories in request"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)